thesauri/mmd-vocabulary.ttl

```diff
@@ -518,7 +525,7 @@
   a skos:Concept ;
   skos:prefLabel "HTTP"@en ;
   skos:broader <https://gcmd.earthdata.nasa.gov/kms/concept/8e33a2dd-df13-4079-8636-391abb5344c6> ;
-  skos:definition "Direct access to the full data file. May require authentication, but should point directly to the data file or a catalogue containing the data."@en .
+  skos:definition "Direct access to the full data file. May require authentication, but should point directly to the data file."@en .
```

Comment:
```suggestion
  skos:definition "Download url for direct access to the complete data file. May require authentication, but should point directly to the data file."@en .
```
Maybe better?

Reply:
in this case, [doc/ch04-md-contrvocabs.adoc](https://github.com/metno/mmd/pull/282/files#diff-853280d26e462218c74b975ced3e50eb04732facac99be314e7ca292952a8778) lines 677 and 678 should be updated as well.